README.md

```diff
@@ -69,6 +69,21 @@ You can install and run the NativeScript CLI on Windows or OS X.
 
 > On Windows systems, you can develop, build, and deploy NativeScript projects that target Android.
 
+### Setup script
+You can run the PowerShell setup script inside a **Command Prompt** against the production branch by running:
```

Comment:
"You can run [...] by running:" sounds a bit odd. 

Consider something like: 


To quickly set up your system for the latest NativeScript CLI, paste the following PowerShell script in the **command prompt** and hit `Enter`:
...
Alternatively, your can paste the following PowerShell setup script in a **Windows PowerShell console** and hit `Enter`: